policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_post_secondary_education_tuition_deduction.py

```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class ar_post_secondary_education_tuition_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas post-secondary education tuition deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/AR1075_2022.pdf#page=1"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.ar.tax.income.deductions.itemized.tuition_deduction
+        tuition_expense = person("qualified_tuition_expenses", period)
+        full_time_college = person("is_full_time_college_student", period)
```

Comment:
need to make a `technical_institution_student` variable (Person level) and use it here